BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -151,6 +330,236 @@ public static string GetVersion(string file, byte[] fileContent, List<int> posit
             return $"{version}.{subVersion:00}.{subsubVersion:000}";
         }
 
+        public static string Get00000001TMPVersion(string firstMatchedString, IEnumerable<string> files)
+        {
+            if (firstMatchedString == null || !File.Exists(firstMatchedString))
+                return string.Empty;
+
+            // This file is present in most, if not all, SafeDisc protected discs. It seems to have very consistent file sizes, only being found to use three different file sizes in it's entire run.
+            FileInfo fi = new FileInfo(firstMatchedString);
+            switch (fi.Length)
+            {
+                // Found in Redump entries 37832 and 66005. 
+                case 20:
+                    return "1.00.025-1.41.001";
+                // Found in Redump entries 30555 and 58573.
+                case 2_048:
+                    return "1.45.011+ (CD) / Cactus Data Shield 300";
+                // Found in Redump entries 11347 and 64255.
+                case 20_482_048:
+                    return "3+ (DVD)";
+                default:
+                    return "Unknown Version (Report this to us on GitHub)";
+            }
+        }
+
+        public static string GetCLCD16Version(string firstMatchedString, IEnumerable<string> files)
+        {
+            if (firstMatchedString == null || !File.Exists(firstMatchedString))
+                return string.Empty;
+
+            // The hash of the file CLCD16.dll is able to provide a broad version range that appears to be consistent, but it seems it was rarely updated so these checks are quite broad.
+            string sha1 = BurnOutSharp.Tools.Utilities.GetFileSHA1(firstMatchedString);
```

Comment:
Please add a `using BurnOutSharp.Tools` to the top of the file so you don't have to use the full path to Utilities every single time.